src/sql/src/plan/statement/ddl/connection.rs

```diff
@@ -398,6 +398,17 @@ impl ConnectionOptionExtracted {
 
                 let tunnel = scx.build_tunnel_definition(self.ssh_tunnel, self.aws_privatelink)?;
 
+                // TODO: Implement proper host verification when using an SSH or PrivateLink tunnel. This
+                // requires a way to modify the TLS configuration created within mysql_async.
+                if matches!(tls_mode, MySqlSslMode::VerifyIdentity)
+                    && matches!(tunnel, Tunnel::Ssh(_) | Tunnel::AwsPrivatelink(_))
```

Comment:
```suggestion
                if matches!((tls_mode, tunnel), (MySqlSslMode::VerifyIdentity, Tunnel::Ssh(_) | Tunnel::AwsPrivatelink(_)))
```

This reads a bit clearer to me, but leave it if you disagree.

Reply:
that is cleaner, will adjust!